statefun-sdk-go/v3/pkg/statefun/egress.go

```diff
@@ -78,7 +78,7 @@ func (k KafkaEgressBuilder) toEgressMessage() (*protocol.FromFunction_EgressMess
 	} else {
 		switch value := k.Value.(type) {
 		case string:
-			_ = StringType.Serialize(&buffer, value)
+			buffer.WriteString(value)
```

Comment:
Does it make sense to size the `buffer` before writing the string?
I can imagine folks sending to en egress large json blobs.

Reply:
The encoding change made this unnecessary, see other comment.